internal/lm/output.go

```diff
@@ -153,6 +153,9 @@ type nodeFeatureObject struct {
 // UpdateNodeFeatureObject creates/updates the node-specific NodeFeature custom resource.
 func (n *nodeFeatureObject) Output(labels Labels) error {
 	nodename := n.nodeConfig.Name
+	if nodename == "" {
+		return fmt.Errorf("required flag %q not set", "node-name")
```

Comment:
if we are passing `"node-name"` as a `string` is it really necessary to set `%q` couldn't we simply just have a flat string `"required flag node-name not set" ` ?